LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019
```

Comment:
Please, mention the name of our organisation "SIESGSTarena" as the LICENSE holder.